src/app/static/src/app/store/baseline/baseline.ts

```diff
@@ -29,12 +32,16 @@ export const initialBaselineState: BaselineState = {
     shapeError: "",
     population: null,
     populationError: "",
-    ready: false
+    ready: false,
+    validatedComplete: false,
+    validatedConsistent: false,
+    baselineError: ""
 };
 
 export const baselineGetters = {
   complete: (state: BaselineState) => {
-      return !!state.country && !!state.iso3 && !!state.shape && !!state.population
+      return state.validatedComplete && state.validatedConsistent &&
+                state.country!! && state.iso3!! && state.shape!! && state.population!!
```

Comment:
Still need to check the individual file states as well, as the combine validate endpoint will return complete and consistent as true if there is an error with an individual file (but no inconsistent / missing files)